modules/openapi-generator/src/main/resources/python/python-experimental/api_client.mustache

```diff
@@ -13,6 +13,9 @@ from six.moves.urllib.parse import quote
 {{#tornado}}
 import tornado.gen
 {{/tornado}}
+{{#hasHttpSignatureMethods}}
+import signing
```

Comment:
Can you change this to:
`from {{packageName}} import signing `

Reply:
sure. With the {{#hasHttpSignatureMethods}}, right?

Reply:
Yup, we need to keep the tag with it